www/include/options/accessLists/groupsACL/listGroupConfig.php

```diff
@@ -38,18 +38,25 @@
 }
 
 include("./include/common/autoNumLimit.php");
+$num = (int) $num;
+$limit = (int) $limit;
```

Comment:
kindly check for negative num and limit

Reply:
all the checks are already done in autoNumLimit.php